src/main/webapp/app/config/constants/firebase.ts

```diff
@@ -98,6 +216,17 @@ export const TX_LEVEL_OPTIONS = [
   TX_LEVELS.LEVEL_NO,
 ];
 
+export const TX_LEVEL_LABELS = {
+  [TX_LEVELS.LEVEL_1]: '1',
+  [TX_LEVELS.LEVEL_2]: '2',
+  [TX_LEVELS.LEVEL_3A]: '3A',
+  [TX_LEVELS.LEVEL_3B]: '3B',
+  [TX_LEVELS.LEVEL_4]: '4',
+  [TX_LEVELS.LEVEL_R1]: 'R1',
+  [TX_LEVELS.LEVEL_R2]: 'R2',
+  [TX_LEVELS.LEVEL_NO]: 'None',
```

Comment:
Historically we use `no`?

Reply:
Got rid of this constant for now. We use 'no' for firebase value, but this constant was for mapping the fb value to a label string that we use on the frontend. (specifically in the filter component, to change the displayed name to 'None')